src/Reactduino.cpp

```diff
@@ -27,6 +37,10 @@ void Reactduino::setup(void)
 void Reactduino::tick(void)
 {
     reaction r;
+    uint32_t now = millis();    // better caching the current time, since it is used in
```

Comment:
I like this change, but not sure we need the comments in the master branch - it's more a thing to document in the merge request comments etc.